src/main/java/reposense/RepoSense.java

```diff
@@ -126,8 +126,11 @@ public static void main(String[] args) {
 
     /**
      * Constructs a list of {@code RepoConfiguration} if {@code cliArguments} is a {@code LocationsCliArguments}.
+     *
+     * @throws InvalidLocationException if all repo locations are invalid.
```

Comment:
Can help change the name of the doc? It is not correct now.

Reply:
Sure. Name changed in the new commit.